src/tracker.ts

```diff
@@ -10,6 +10,7 @@ export interface Entry {
     startTime: string;
     endTime: string;
     subEntries: Entry[];
+    collapsed?: number;
```

Comment:
Why is collapsed a number and not a boolean?

Reply:
Honestly, I was just making it a number so that it would be smaller in the serialized JSON. I can switch it to a boolean.

Reply:
Resolved by c587f8389016084f66155b48e77956228f9420cb